docs/_posts/2024-07-08-v4.11.5.md

```diff
@@ -30,21 +30,21 @@ We thank all organizations and individuals who contributed to this release, from
 Special thanks to everyone who contributed code to implement enhancements and fix defects:  
 [@2000rosser]
 
-###### dependency-track-apiserver.jar
+###### [dependency-track-apiserver.jar](https://github.com/DependencyTrack/dependency-track/releases/download/4.11.5/dependency-track-apiserver.jar)
 
 | Algorithm | Checksum                                                         |
 |:----------|:-----------------------------------------------------------------|
 | SHA-1     | 8fd45ea6ae725e8e7dac59ec9d471fcdaeb42c6d                         |
 | SHA-256   | c39c15849cbb7dd19833ea689c20aaf92bc9f6965b758961e1d2a01a2b09f86f |
 
-###### dependency-track-bundled.jar
+###### [dependency-track-bundled.jar](https://github.com/DependencyTrack/dependency-track/releases/download/4.11.5/dependency-track-bundled.jar)
 
 | Algorithm | Checksum                                                         |
 |:----------|:-----------------------------------------------------------------|
 | SHA-1     | eba6cbaa6c2da9ffb295da83ed39af68ff4130a8                         |
 | SHA-256   | 7ebb11573b2a59084ed98fe92d363240c910dc7b5aa7ebeda64bee7d47089d9a |
 
-###### frontend-dist.zip
+###### [frontend-dist.zip](https://github.com/DependencyTrack/dependency-track/releases/download/4.11.5/frontend-dist.zip)
```

Comment:
Frontend links should point to the `DependencyTrack/frontend` repository. The current links yield a 404.

```suggestion
###### [frontend-dist.zip](https://github.com/DependencyTrack/frontend/releases/download/4.11.5/frontend-dist.zip)
```